README.md

```diff
@@ -79,6 +79,37 @@ Save the CLAUDE.md file at the root of the repository. Alternatively, merge the
 curl https://raw.githubusercontent.com/dagger/container-use/main/rules/agent.md >> CLAUDE.md
 ```
 
+### [Amazon Q Developer CLI chat](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-chat.html)
+
+Add this container-use MCP config to `~/.aws/amazonq/mcp.json`:
+
+```json
+{
+  "mcpServers": {
+    "container-use": {
+      "command": "cu",
+      "args": [
+        "stdio"
+      ],
+      "env": {},
+      "timeout": 60000
+    }
+  }
+}
+```
+
+Save the agent instructions for Container Use to your project root at `./.amazonq/rules/container-use.md`:
+
+```sh
+mkdir -p ./.amazonq/rules && curl https://raw.githubusercontent.com/dagger/container-use/main/rules/agent.md > .amazonq/rules/container-use.md
```

Comment:
nit: This can be done with a single command as

`curl --create-dirs -o .amazonq/rules/container-use.md https://raw.githubusercontent.com/dagger/container-use/main/rules/agent.md`